save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OwnerView.kt

```diff
@@ -0,0 +1,212 @@
+/**
+ * A view with owner details
+ */
+
+package org.cqfn.save.frontend.components.views
+
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.cqfn.save.entities.Project
+import org.cqfn.save.frontend.components.basic.privacySpan
+import org.cqfn.save.frontend.components.tables.tableComponent
+import org.cqfn.save.frontend.utils.*
+import org.w3c.fetch.Headers
+import react.*
+import react.dom.*
+import react.table.columns
+import csstype.Position
+import csstype.Top
+import csstype.Left
+import csstype.TextAlign
+import kotlinx.html.InputType
+import kotlinx.html.hidden
+import kotlinx.html.js.onChangeFunction
+import org.cqfn.save.domain.ImageInfo
+import org.w3c.dom.HTMLInputElement
+import org.w3c.dom.asList
+import org.w3c.xhr.FormData
+
+/**
+ * `Props` retrieved from router
+ */
+@Suppress("MISSING_KDOC_CLASS_ELEMENTS")
+external interface OwnerProps : PropsWithChildren {
+    var owner: String
+}
+
+/**
+ * [State] of project view component
+ */
+external interface OwnerViewState : State {
+
+    /**
+     * Flag to handle uploading a file
+     */
+    var isUploading: Boolean
+
+    /**
+     * Image to owner avatar
+     */
+    var image: ImageInfo?
```

Comment:
Now you are storing all image data in memory along with all other page data. This is incorrect. What we wanted is you write something like `<img src="backend/images/avatars/$owner"/>` and the browser loads the image from the provided URL.